exec/java-exec/src/test/java/org/apache/drill/exec/planner/rm/TestMemoryCalculator.java

```diff
@@ -59,6 +59,7 @@
 
   private static final long DEFAULT_SLICE_TARGET = 100000L;
   private static final long DEFAULT_BATCH_SIZE = 16*1024*1024;
+  private static final String ENABLE_QUEUE = "drill.exec.queue.embedded.enable";
```

Comment:
Minor comment. You can use: `EmbeddedQueryQueue.ENABLED` instead of again defining it here.

Reply:
I have updated the test case.